### PR TITLE
Improve admin tags

### PR DESCRIPTION
**Pull request by netagonen**
Changes for this PR:  
- improve filters area
- After an action on the edit page, go back to same tag on the tag overview
- Move notifications relatively to user location

Questions for this PR: 
1. I left out the validation that tag has tagging (and therefor a profile). Adding this line of code:
`validates :taggings, :presence => true ` 
caused 143 test failures with this message: 
`ActiveRecord::RecordInvalid: Gültigkeitsprüfung ist fehlgeschlagen: Taggings muss ausgefüllt werden`
because every time we create a tag in the specs we also need to have a tagging. 
This might deserve a total different issue :) 
What do you think? 
2. The categories filter: all options are radio buttons except the uncategorized option, which is a check-box. Do we want to do something about it? It seems a bit weird.. but on the other hand changing it means back-end work and I am not sure it is worth it. What do you think?

### CODE REVIEW

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -141,6 +143,8 @@ h1, h4 {
   font-family: $sans-serif;
   padding: $base-spacing-unit;
   text-align: center;
+	position: fixed;
+	width: 100%;
```
Comment (tyranja) [inline app/assets/stylesheets/custom.scss]: Hm, indentation seems to be wrong?

**Comment by tyranja** [comment]
To the first question, yes it would make sense to create an own ticket and PR for the validation of the tags!

File: app/assets/stylesheets/custom.scss (hunk 1 above)
Comment (netagonen) [reply app/assets/stylesheets/custom.scss]: fixed

**Comment by netagonen** [comment]
Yes, I thought this is the change that needed to be done. So a bit more work then :)

**Comment by netagonen** [comment]
Also done!

**Comment by tyranja** [comment]
That is great!!!! Thank you!

**Comment by tyranja** [comment]
I have solved the merge conflicts.

**Pull request merged**